src/main/java/ch/njol/skript/events/EvtWorld.java

```diff
@@ -0,0 +1,97 @@
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.bukkit.event.world.WorldInitEvent;
+import org.bukkit.event.world.WorldLoadEvent;
+import org.bukkit.event.world.WorldSaveEvent;
+import org.bukkit.event.world.WorldUnloadEvent;
+import org.bukkit.event.world.WorldEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class EvtWorld extends SkriptEvent {
+
+	static {
+		// World Save Event
+		Skript.registerEvent("World Save", EvtWorld.class, WorldSaveEvent.class, "world sav(e|ing) [of %-worlds%]")
+				.description("Called when a world is saved to disk. Usually all worlds are saved simultaneously, but world management plugins could change this.")
+				.examples(
+					"on world save of \"world\":",
+					"\tbroadcast \"The world %event-world% has been saved\"")
+				.since("1.0, INSERT VERSION (defining worlds)");
+
+		// World Init Event
+		Skript.registerEvent("World Init", EvtWorld.class, WorldInitEvent.class, "world init[ialization] [of %-worlds%]")
+				.description("Called when a world is initialized. As all default worlds are initialized before",
+					"any scripts are loaded, this event is only called for newly created worlds.",
+					"World management plugins might change the behaviour of this event though.")
+				.examples("on world init of \"world_the_end\":")
+				.since("1.0, INSERT VERSION (defining worlds)");
+
+		// World Unload Event
+		Skript.registerEvent("World Unload", EvtWorld.class, WorldUnloadEvent.class, "world unload[ing] [of %-worlds%]")
+				.description("Called when a world is unloaded. This event will never be called if you don't have a world management plugin.")
+				.examples(
+					"on world unload:",
+					"\tbroadcast \"the %event-world% has been unloaded!\"")
+				.since("1.0, INSERT VERSION (defining worlds)");
+
+		// World Load Event
+		Skript.registerEvent("World Load", EvtWorld.class, WorldLoadEvent.class, "world load[ing] [of %-worlds%]")
+				.description("Called when a world is loaded. As with the world init event, this event will not be called for the server's default world(s).")
+				.examples(
+					"on world load of \"world_nether\":",
+					"\tbroadcast \"The world %event-world% has been loaded!\"")
+				.since("1.0, INSERT VERSION (defining worlds)");
+	}
+
+	@Nullable
+	private Literal<World> worlds;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult) {
+		worlds = (Literal<World>) args[0];
+		if (worlds != null && worlds.getAnd()) {
+			Skript.error("An \"and\" list cannot be used in this event as multiple worlds cannot be called at the same time");
+			return false;
+		}
```

Comment:
This prevents you from doing `on world load of "world_nether"` because it's an and list of 1, I think.
![image](https://github.com/SkriptLang/Skript/assets/10354869/78b2859b-9d03-4963-9deb-60b49a397dd6)
![image](https://github.com/SkriptLang/Skript/assets/10354869/a2f3cfc3-2891-4db2-97f8-ef072ade4606)

Reply:
![image](https://github.com/SkriptLang/Skript/assets/10354869/2644a931-8c5c-4d5a-9948-795e677707a9)
A solution could be what I did for evtgrow, where you force And lists to be Or.

Reply:
> ![image](https://github.com/SkriptLang/Skript/assets/10354869/2644a931-8c5c-4d5a-9948-795e677707a9)
> A solution could be what I did for evtgrow, where you force And lists to be Or.

Wonder if something like that is better for skript to eventually force. Iirc my set and wasn't needed as internally it's an or

Reply:
So am back at this for first thing of the new year and I'm honestly unsure if I should bother keeping this error. For one `and/or` both work wording wise fine to me. The whole reason it's even done is `on save of "World" and "World_Nether"` being read as called when both worlds are saved rather than one or the other but skript doesn't ever take this into account.

![image](https://github.com/SkriptLang/Skript/assets/81952476/bda48074-7731-46e9-bcd1-a47d3e1830b9)

Reply:
Alright so after a bit of looking, I decided to do what EvtClick does which should of been the suggested path tbh. When the defined areas are single i.e. `%world%` instead of `%worlds%` you're still able to do patterns like `world save [of %world%]` which allows `world` and `world or world_nether` will error with `world and world_the_end` without me handling it.

New commits will be done shortly

Reply:
> For one `and/or` both work wording wise fine to me. 

Yes that was my suggestion, which I did for evtgrow. You allow the user to write either, but you act as if it is OR either way.